.gitattributes

```diff
@@ -0,0 +1,11 @@
+* text=auto
```

Comment:
Is this line necessary?

Reply:
Well I think we might [keep it](https://stackoverflow.com/questions/21472971/what-is-the-purpose-of-text-auto-in-gitattributes-file). Thank you!